include/arbor/lif_cell.hpp

```diff
@@ -3,7 +3,7 @@
 namespace arb {
 
 // Model parameteres of leaky integrate and fire neuron model.
-struct lif_cell_description {
+struct lif_cell {
```

Comment:
OK. I see, each cell type/description should have an `X_cell` description type in the public api.